include/boost/math/distributions/von_mises.hpp

```diff
@@ -0,0 +1,842 @@
+//  Copyright John Maddock 2006, 2007.
+//  Copyright Paul A. Bristow 2006, 2007.
+
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_STATS_VON_MISES_HPP
+#define BOOST_STATS_VON_MISES_HPP
+
+// https://en.wikipedia.org/wiki/Von_Mises_distribution
+// From MathWorld--A Wolfram Web Resource.
+// http://mathworld.wolfram.com/VonMisesDistribution.html
+
+#include <boost/math/distributions/fwd.hpp>
```

Comment:
Can we remove the forward header? @jzmaddock  or @pabristow might have a better idea that I about why this is necessary but in general I find it causes problems.